api/block/routes.go

```diff
@@ -0,0 +1,210 @@
+package block
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/ElrondNetwork/elrond-go/api/errors"
+	"github.com/ElrondNetwork/elrond-go/api/shared"
+	"github.com/ElrondNetwork/elrond-go/api/wrapper"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	getBlockByNonceEndpoint        = "/block/by-nonce/:nonce"
+	getBlockByNonceWithTxsEndpoint = "/block/by-nonce/:nonce/transactions"
+	getBlockByHashEndpoint         = "/block/by-hash/:hash"
+	getBlockByHashWithTxsEndpoint  = "/block/by-hash/:hash/transactions"
+)
+
+// BlkService interface defines methods that can be used from `elrondFacade` context variable
```

Comment:
I used BlkService instead of BlockService because i got a warning (name of interface starts with the package name). But I can to rename it  with your suggestion.